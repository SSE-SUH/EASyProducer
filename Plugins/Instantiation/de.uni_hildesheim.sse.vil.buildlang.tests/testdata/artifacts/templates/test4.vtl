template test4(Configuration config, XmlFileArtifact target) {

    def main(Configuration config, XmlFileArtifact target) {
        target.setOmitXmlDeclaration(true);
        '<root xml="true" open="false" adaptive="true"/>'
    }

}