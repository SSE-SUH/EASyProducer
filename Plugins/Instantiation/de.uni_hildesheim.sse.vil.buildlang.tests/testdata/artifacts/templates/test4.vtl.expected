<root xml="true" open="false" adaptive="true"/>